dod-ccpo/atat
# [AT-5815] Generate SAML IDP as part of deployment

Run the SAML IDP creation script and pass it into the environment of the application.

## How to test

- Go to an existing provisioner container instance
- Clone down the branch
- Run provisioner.py
- You’d then have to jump into the bastion
- Clone the new deploy/azure/secret_provider.yml overlay
- `kubectl apply`
-  that as well
- Then you should be able to see in the `kubectl describe`
